src/sentry/api/base.py

```diff
@@ -108,17 +108,16 @@ def create_audit_entry(self, request, **kwargs):
             **kwargs
         )
 
+        extra = {
+            'entry_id': entry.id,
+            'actor_label': entry.actor_label
+        }
         if entry.actor_id:
-            logger.bind(actor_id=entry.actor_id)
+            extra['actor_id'] = entry.actor_id
         if entry.actor_key_id:
-            logger.bind(actor_key_id=entry.actor_key_id)
+            extra['actor_key_id'] = entry.actor_key_id
 
-        logger.info(
-            name='sentry.audit.entry',
```

Comment:
Yeah, I think it'd be valuable to be distinct about audit stuff to a different logger so it's easier to search for.